x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -555,8 +556,10 @@ const allowlistEventFields: AllowlistFields = {
   data_stream: true,
   ecs: true,
   elastic: true,
-  // behavioral protection re-nests some field sets under events.*
+  // behavioral protection re-nests some field sets under events.* (< 7.15)
   events: allowlistBaseEventFields,
+  // behavioral protection re-nests some field sets under Events.* (>=7.15)
+  Events: allowlistBaseEventFields,
```

Comment:
Looks good. Please help out with data engineering by keeping the data dictionary up to date in our pro cloud repo.